### PR TITLE
Add "sum" filter

### PR DESCRIPTION
**Pull request by lazopm**
http://jinja.pocoo.org/docs/dev/templates/#sum

### CODE REVIEW

File: src/filters.js [hunk 1]
```diff
@@ -358,7 +358,25 @@ var filters = {
 
         return res;
     },
-
+    
+    sum: function(arr, start, attr) {
```
Comment (carljm) [inline src/filters.js]: looks like these arguments are in the opposite order from the jinja version

File: src/filters.js [hunk 2]
```diff
@@ -358,7 +358,25 @@ var filters = {
...
+        }
+        
+        if(attr) {
+            arr = lib.map(arr, function(v) {
+                return v[attr];
+            });
+        }
+        
+        for(var i = 0; i < arr.length; i++) {
+            sum += arr[i];
```
Comment (carljm) [inline src/filters.js]: This var seems to be undeclared. And also you aren't using the `start` value anywhere. Were those intended to be the same var?

**Comment by carljm** [comment]
Made a couple comments, and tests are failing (due to the same issue pointed out in my comments). But with the comments addressed, this looks good, thanks!

**Comment by lazopm** [comment]
@carljm Does the tests look OK?

**Comment by carljm** [comment]
Looks good, thanks! Merging.

**Pull request merged**